console/Command.php

```diff
@@ -109,17 +116,23 @@ protected function _init() {
 		if (!is_object($this->request) || !$this->request->params) {
 			return;
 		}
-		$this->response = $this->_config['response'];
 
-		if (!is_object($this->response)) {
-			$this->response = $this->_instance('response', $this->response);
-		}
 		$default = array('command' => null, 'action' => null, 'args' => null);
 		$params = array_diff_key((array) $this->request->params, $default);
 
 		foreach ($params as $key => $param) {
+			if($key == 'no-color') {
```

Comment:
Isn't no-color mapped automatically to noColor? We might to change the console routing if slugged multi word args are not possible.